pkg/gits/helpers.go

```diff
@@ -590,12 +600,16 @@ func ForkAndPullRepo(gitURL string, dir string, baseRef string, branchName strin
 		}
 	}
 
+	log.Logger().Info("foo")
+
 	if stashed {
 		err = gitter.StashPop(dir)
 		if err != nil && !IsNoStashEntriesError(err) { // Ignore no stashes as that's just because there was nothing to stash
 			return "", "", nil, nil, errors.Wrapf(err, "unable to pop the stash")
 		}
 	}
+
+	log.Logger().Info("bar")
```

Comment:
ditto

Reply:
removed. Had it there to for debugging purposes